src/rpc/chain.ts

```diff
@@ -1017,11 +1019,22 @@ export class ChainRpc {
                 `Expected the second argument of getPendingTransactions to be a non-negative integer but found ${to}`
             );
         }
+        if (futureIncluded !== true && futureIncluded !== false) {
```

Comment:
To be consistent with others, `typeof futureIncluded === "boolean"` would be better.